### PR TITLE
[SCI] Clarify git metadata upload usage

### PR DESCRIPTION
**Pull request by Letiste**
### What does this PR do? What is the motivation?

Some customers are confused on how to use the git-metadata upload command. This PR clarifies (hopefully) the usage.

Before:
image

After:
image
 

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/guide/source-code-integration.md [hunk 1]
```diff
@@ -203,10 +203,12 @@ Setting up the GitHub integration also allows you to see inline code snippets in
 Repositories on self-hosted instances or private URLs are not supported out-of-the-box by the Source Code Integration. To enable this feature, <a href="/help">contact Support</a>.
 </div>
 
-To link telemetry to your source code, Datadog collects metadata for every commit SHA from your Git repository with the [`datadog-ci git-metadata upload`][1] command.
+To link telemetry to your source code, you can upload your repository metadata with the [`datadog-ci git-metadata upload`][1] command.
 
 When you run `datadog-ci git-metadata upload` within a Git repository, Datadog receives the repository URL, the commit SHA of the current branch, and a list of tracked file paths.
 
+This command needs to be run for every commit you need to be synched to Datadog.
```
Comment (rpelliard) [inline content/en/integrations/guide/source-code-integration.md:210]:
```suggestion
This command needs to be run for every commit you need to be synchronized to Datadog.
```

**Review by rpelliard — APPROVED** [review]

File: content/en/integrations/guide/source-code-integration.md (hunk 1 above)
Comment (rpelliard) [reply content/en/integrations/guide/source-code-integration.md:210]: (Or `synced` but I think there is no `h` in the short version)

Comment (Letiste) [reply content/en/integrations/guide/source-code-integration.md:210]: Let's pick `synchronized` 😅

File: content/en/integrations/guide/source-code-integration.md [hunk 2]
```diff
@@ -203,10 +203,12 @@ Setting up the GitHub integration also allows you to see inline code snippets in
 Repositories on self-hosted instances or private URLs are not supported out-of-the-box by the Source Code Integration. To enable this feature, <a href="/help">contact Support</a>.
 </div>
 
-To link telemetry to your source code, Datadog collects metadata for every commit SHA from your Git repository with the [`datadog-ci git-metadata upload`][1] command.
+To link telemetry to your source code, you can upload your repository metadata with the [`datadog-ci git-metadata upload`][1] command.
 
 When you run `datadog-ci git-metadata upload` within a Git repository, Datadog receives the repository URL, the commit SHA of the current branch, and a list of tracked file paths.
 
+This command needs to be run for every commit you need to be synchronized to Datadog.
```
Comment (brett0000FF) [inline content/en/integrations/guide/source-code-integration.md:210]:
```suggestion
Run this command for every commit that you need to be synchronized with Datadog.
```

**Review by brett0000FF — APPROVED** [review]

**Comment by brett0000FF** [comment]
Thank you!

**Pull request merged**